deploy/Chart/templates/controller/deployment.yaml

```diff
@@ -48,6 +48,9 @@ spec:
           name: metrics
           protocol: TCP
         {{- end }}
+        - containerPort: 3000
+          name: healthz
```

Comment:
This is pretty clear btw. By calling the port `healthz` I immediately understand what it's for. Do you think we could the same in our configuration files?